docs/admin/logging/overview.md

```diff
@@ -130,6 +131,10 @@ By default, Docker uses the first 12 characters of the container ID to tag log m
 Refer to the [log tag option documentation](log_tags.md) for customizing
 the log tag format.
 
+`syslog-format` configures syslog protocol version to use when logging,
+Allowed value is rfc5424 in order to use RFC5424 syslog protocol. 
+Default is RFC3164 incompatible syslog protocol.
```

Comment:
@thaJeztah  please take a look at 
https://github.com/RackSec/srslog/blob/master/formatter.go in commets it says exactly what is currently supported.

Note that default is "confusing" it defaults to golang syslog package not to the previous implementation that you can observe here 
https://github.com/RackSec/srslog/blob/68fa06c8afc0374f78d4be6107081b17a2c349c9/srslog_unix.go#L40
and which is similar to unixformatter